google-cloud-spanner/src/main/java/com/google/cloud/spanner/Instance.java

```diff
@@ -61,6 +62,18 @@ public Builder setDisplayName(String displayName) {
       return this;
     }
 
+    @Override
+    public Builder setUpdateTime(Timestamp updateTime) {
```

Comment:
Should we make it protected?

Reply:
I would opt for package-private if possible. Otherwise protected. This is not something that a client should be able to set.

Reply:
package-private modifier seems like a slightly better fit as providing override option outside the package will not be of much use.